client/app/hosting/framework/envvar/hosting-framework-envvar.service.js

```diff
@@ -0,0 +1,134 @@
+angular.module("services").service(
+    "HostingFrameworkEnvvar",
+    class HostingFrameworkEnvvar {
+
+        /**
+         * @constructs HostingFrameworkEnvvar
+         * @param $q
+         * @param OvhHttp
+         * @param Hosting
+         */
+        constructor ($q, OvhHttp, Hosting) {
```

Comment:
You could sort the service names alphabetically but I'm being pernickety there 😃